zproject/test_settings.py

```diff
@@ -1,6 +1,8 @@
 from __future__ import absolute_import
-from .settings import *
 import os
+if os.getenv("EXTERNAL_HOST") is None:
```

Comment:
Let's add a comment here explaining what's going on with this file and why we need to put this hack here :)